src/Aspire.Hosting/Dashboard/ConsoleLogPublisher.cs

```diff
@@ -19,6 +19,7 @@ internal IAsyncEnumerable<IReadOnlyList<(string Content, bool IsErrorMessage)>>?
         {
             ExecutableSnapshot executable => SubscribeExecutable(executable),
             ContainerSnapshot container => SubscribeContainer(container),
+            GenericResourceSnapshot => GenericResourceSnapshot.GetLogsEnumerable(),
```

Comment:
FYI this code is about to change. @karolz-ms is updating it to get logs from DCP. Need to think about how custom resources work in that world.

Reply:
Should work the same. I have a hack here https://github.com/dotnet/aspire/blob/321d0a20690c5e6a9cfda8558044b26769bc5e36/src/Aspire.Hosting/Dashboard/GenericResourceSnapshot.cs#L26-L40. This would OOM eventually though so would need a circular buffer to cache old results.